extract.py

```diff
@@ -1,3 +1,8 @@
+"""
+Extracts the information from C files and
+creates a call graph using igraph.
```

Comment:
Dumb question: what's the output format?

Reply:
That is actually important. Igraph can be used to generate multiple output formats but atm, I don't store the graph on the fs as it is generated fast enough from the c files. So at the moment I save only the svg images.

Reply:
I might be being silly but... what's `Igraph`? :p And is this script producing output that goes straight into Igraph?

Reply:
This is igraph: https://igraph.org/ and it is not a separate program it is a library for analyzing graphs.